test/e2e/tests/account/snap-account-signatures.spec.ts

```diff
@@ -18,7 +18,7 @@ import SnapSimpleKeyringPage from '../../page-objects/pages/snap-simple-keyring-
 import TestDapp from '../../page-objects/pages/test-dapp';
 
 describe('Snap Account Signatures @no-mmi', function (this: Suite) {
-  this.timeout(150000); // This test is very long, so we need an unusually high timeout
+  this.timeout(200000); // This test is very long, so we need an unusually high timeout
```

Comment:
Could you explain why this timeout is needed in general? I see that this is a multiple test scenario in a single test, and it being flaky irrespective of the timeout itself. Could we enhance this in another way, or should it be addressed in a separate ticket then incase then this change made not be needed is my thought.

Reply:
This test is very long. For each account signature flow type, it executes all the signature types. As we have encountered timeout error in CI, after the PR is merged, we need to monitor the CI execution results. If we still have the error, we may consider breaking this test file into several test files.